packages/next/server/app-render.tsx

```diff
@@ -209,7 +213,10 @@ function patchFetch(ComponentMod: any) {
 
   const origFetch = (global as any).fetch
 
-  ;(global as any).fetch = async (init: any, opts: any) => {
+  ;(global as any).fetch = async (
+    url: RequestInfo,
+    opts: RequestInit & { next?: NextFetchRequestConfig }
```

Comment:
We need to augment the global type as well which we can probably add in https://github.com/vercel/next.js/blob/canary/packages/next/types/global.d.ts